site/themes/s2b_hugo_theme/layouts/partials/cal/dayjs.html

```diff
@@ -4,10 +4,13 @@
 <script src="/lib/dayjs/dayjs.min.js"></script>
 <script src="/lib/dayjs/plugin/utc.js"></script>
 <script src="/lib/dayjs/plugin/customParseFormat.js"></script>
+<script src="/lib/dayjs/plugin/duration.js"></script>
+<script src="/lib/dayjs/plugin/js"></script>
```

Comment:
Typo? Doesn't seem to refer to a real file.

Reply:
Yeah, must have been a copy/paste thing. Didn't catch it